src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_IntegrationTests.cs

```diff
@@ -10,6 +10,7 @@
 public partial class ThreadPoolBoundHandleTests
 {
     [Fact]
+    [PlatformSpecific(TestPlatforms.Windows)] // ThreadPoolBoundHandle.BindHandle is not supported on Unix
```

Comment:
FYI, if all tests in a test class are specific to the same platform, you can put the PlatformSpecific attribute on the test class once rather than on each individual test method.